CRM/ACL/BAO/ACL.php

```diff
@@ -258,7 +226,18 @@ public static function getAllByContact($contact_id) {
 
     /* First, the contact-specific ACLs, including ACL Roles */
     if ($contact_id) {
-      $result += self::getACLs((int) $contact_id);
+      $query = " SELECT acl.*
+      FROM civicrm_acl acl
+      WHERE   acl.entity_table   = 'civicrm_contact'
```

Comment:
how does this even work?

Reply:
@seamuslee001 in terms of 'when would entity_table be equal to 'civicrm_contact'? - my suspicion is that it has not been possible for a decade - I'm still trying to test that theory